portal/src/graphql/portal/SubscriptionPlanCard.tsx

```diff
@@ -158,7 +158,7 @@ export function UsagePriceTag(props: UsagePriceTagProps): React.ReactElement {
 }
 
 export interface CTAProps {
-  variant: "subscribe" | "upgrade" | "downgrade" | "current";
+  variant: "subscribe" | "upgrade" | "downgrade" | "current" | "non-applicable";
```

Comment:
The visual appearance of non-applicable is a bit subtle as it has a "-" label only. What about rendering a disabled subscribe variant to represent non-applicable?